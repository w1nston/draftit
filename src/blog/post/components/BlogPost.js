import React, { Fragment } from 'react';

// TODO: fetch data from persistence layer...
const BlogPost = props => (
  <Fragment>
    <h2>{props.match.params.title}</h2>
    <div>TODO: Add content...</div>
  </Fragment>
);

export default BlogPost;
