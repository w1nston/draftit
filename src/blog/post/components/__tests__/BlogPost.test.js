import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import BlogPost from '../BlogPost';

describe('BlogPost', () => {
  it('renders', () => {
    const match = { params: { title: 'Title' } };
    const component = shallow(<BlogPost match={match} />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
