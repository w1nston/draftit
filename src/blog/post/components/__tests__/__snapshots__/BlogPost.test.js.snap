// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogPost renders 1`] = `
<React.Fragment>
  <h2>
    Title
  </h2>
  <div>
    TODO: Add content...
  </div>
</React.Fragment>
`;
